### DISCUSSION TITLE
Ability to set defaults for `loaders` `versions` and `release channel`, per project, rather than setting them on each version

### DISCUSSION THREAD

**Discussion by RhythmicSys** [modrinth/.github#67] [Feature requests]
### Is your feature request related to a problem? Please describe.

I have a ton of projects running at one time, I literally do not remember which API version each one is on, and every time I do a small bugfix and then have to check again, it's very frustrating. 

### Describe the solution you'd like

I feel like being able to set/modify defaults for the `loaders` `minecraft versions` and `release channel` would be extremely helpful. 
I think the ability to add on/modify them on release is still useful, but having to input it every time seems extremely redundant.

### Describe alternatives you've considered

I have not considered other alternatives as of this moment

### Additional context

![image](https://user-images.githubusercontent.com/45906780/198188774-1f5ba85b-ac02-4d17-b45f-5aedcffe9728.png)
I did not actually change the API version on any of these, I just forgot

**Comment by Arbee4ever** [comment]
You can use [minotaur](https://github.com/modrinth/minotaur) to easily configure and publish the new Version. This also allows you to set versions and Loaders for each project.

**Comment by RhythmicSys** [comment]
I don't use gradle for my projects, though

**Comment by Arbee4ever** [comment]
Then there is [this](https://github.com/Kir-Antipov/mc-publish)

**Comment by RhythmicSys** [comment]
It was mainly a suggestion as that's the one thing from other plugin platforms (Spigot specifically) I found very useful, I don't find having to go through GH actions or use an additional plugin nearly as intuitive or helpful.

**Comment by Arbee4ever** [comment]
I just wanted to provide you ways you can achieve that right now, without waiting for modrinth to implement something :D

**Comment by RhythmicSys** [comment]
thanks :3
